cmake/ctest/drivers/lightsaber/cron_driver.sh

```diff
@@ -8,36 +8,36 @@ echo
 # TrilinosDriver settings:
 #
 
-setenv TDD_PARALLEL_LEVEL 2
+export TDD_PARALLEL_LEVEL=2
 
 # Trilinos settings:
 #
 
 # Submission mode for the *TrilinosDriver* dashboard
-setenv TDD_CTEST_TEST_TYPE Nightly
+export TDD_CTEST_TEST_TYPE=Nightly
 
 
 # Machine specific environment
 #
 
-setenv TDD_HTTP_PROXY "http://wwwproxy.sandia.gov:80"
-setenv TDD_HTTPS_PROXY "https://wwwproxy.sandia.gov:80"
-setenv http_proxy "http://wwwproxy.sandia.gov:80"
-setenv https_proxy "https://wwwproxy.sandia.gov:80"
-setenv TDD_FORCE_CMAKE_INSTALL 1
-setenv TDD_DEBUG_VERBOSE 1
+export TDD_HTTP_PROXY="http://wwwproxy.sandia.gov:80"
+export TDD_HTTPS_PROXY="https://wwwproxy.sandia.gov:80"
+export http_proxy="http://wwwproxy.sandia.gov:80"
+export https_proxy="https://wwwproxy.sandia.gov:80"
```

Comment:
`wwwproxy.sandia.gov` should be `proxy.sandia.gov`.

Reply:
You're right, but since we don't actually use external network connections, it doesn't matter.

Reply:
just delete?